.travis.yml

```diff
@@ -1,9 +1,13 @@
+dist: jammy
```

Comment:
python 3.10 and 3.11 aren't available on the current default ubuntu install, so now I manually set the version to use.